app/views/admin/conversations/show.html.haml

```diff
@@ -7,30 +10,36 @@
 
 = render 'admin/nav', {active: 'conversations'}
 
-%p#notice= notice
-
-%h4 Conversation
+%h4
+  Conversation #{@conversation.id}
 = render partial: 'admin/conversations/table', locals: {conversations: [@conversation], include_show: false}
-%p
-  = link_to 'Back to Conversations', admin_conversations_url
-%h4 Messages
+
+%h4
+  Messages
+
 %table.admin
   %thead
     %tr
       %th Time
-      %th From
-      %th To
+      %th Sent By
       %th Message
 
   %tbody
     - @conversation.messages.order('created_at asc').each do |message|
       %tr
         %td= message.created_at.localtime.strftime('%m/%d %l:%M%P')
-        %td= message.from
-        %td= message.to
+
+        %td
+          - if message.from.phony_formatted == @conversation.from_phone.phony_formatted
```

Comment:
I implemented a Message#is_from_voter? that checks this by matching ridezone # to conversation to_phone, but this works too. Either way, recommend moving this to the model check and not do in the view.